public/index.html

```diff
@@ -73,72 +73,71 @@
           });
       };
     </script>
-    <script>
-      document.addEventListener("DOMContentLoaded", () => {
-        const defineCustomElementsFromTemplates = () => {
-          const templates = document.querySelectorAll("template");
-
-          templates.forEach(template => {
-            const templateId = template.id;
-
-            class CustomElement extends HTMLElement {
-              constructor() {
-                super();
-
-                // Instantiate the body from the template here. It's suggested
-                // to do it in connectedCallback instead, but when nodes are
-                // moved around things get duplicated, so that's not really
-                // suitable.
-                this.copyFromTemplate();
-              }
 
-              // Take the template content and create a body for this
-              // instantiation of the customelement from it.
-              // 1. Replace the data-content node in the template with the children of the customelement
-              // 2. Use the customElement's attributes and replace placeholder-* attributes in the template
-              copyFromTemplate() {
-                let body = template.content.cloneNode(true);
-
-                // Replace attributes
-                Array.from(this.attributes).forEach(attr => {
-                  const placeholderAttribute = `placeholder-${attr.name}`;
-                  let count = 0;
-                  body
-                    .querySelectorAll(`[${placeholderAttribute}]`)
-                    .forEach(element => {
-                      count = count + 1;
-                      element.setAttribute(attr.name, attr.value);
-                      element.removeAttribute(placeholderAttribute);
-                    });
-                  if (count > 0) {
-                    this.removeAttribute(attr.name);
-                  }
-                });
+    <script defer type="module">
```

Comment:
Replaced `document.addEventListener("DOMContentLoaded", () => ` with `defer`.  It accomplishes the same functionality.

Reply:
Great, thanks!